camel/types/openbb_types.py

```diff
@@ -0,0 +1,61 @@
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ========= Copyright 2023-2024 @ CAMEL-AI.org. All Rights Reserved. =========
+
+from typing import Any, Protocol, Union
+
+
+class OpenBBBaseProtocol(Protocol):
+    """Protocol defining the base interface for OpenBB SDK functionality."""
```

Comment:
docstring format, same for other parts 

```suggestion
    r"""Protocol defining the base interface for OpenBB SDK functionality."""
```

Reply:
Thanks @Wendong-Fan for your comment, I just updated the docstring to make sure it strictly follows Camel's standard.